translations/ui-circulation/en.json

```diff
@@ -469,5 +469,6 @@
   "permission.settings.overdue-fines-policies": "Settings (Circ): Can create, edit and remove overdue fine policies",
   "permission.settings.lost-item-fees-policies": "Settings (Circ): Can create, edit and remove lost item fee policies",
   "permission.settings.view-overdue-fines-policies": "Settings (Circ): Can view overdue fine policies",
-  "permission.settings.view-lost-item-fees-policies": "Settings (Circ): Can view lost item fee policies"
+  "permission.settings.view-lost-item-fees-policies": "Settings (Circ): Can view lost item fee policies",
+  "permission.settings.titleLevelRequests": "Settings (Circ): Can modify title level requests settings"
```

Comment:
it looks a little bit weird that we have camelcase and snakecase in translations. Looks like different modules have different agreements.

Reply:
@pavelspichonak  How i understood we don't have rules that describe name convention for current case. On my opinion using camel case here will be better and this will be consistency with naming that we use in translation files.
CC: @Stanislau-Snapok